charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -31,6 +41,16 @@ sidecars:
       pullPolicy: IfNotPresent
       repository: k8s.gcr.io/sig-storage/csi-attacher
       tag: "v3.4.0"
+    # Tune leader lease election for csi-attacher.
+    # Leader election is on by default.
+    leaderElection:
+      enabled: true
+      # Optional values to tune lease beaviour.
```

Comment:
final nit: behavior. Also, please squash the commits before we merge this in. Other than that, lgtm. I will approve this PR once another maintainer has reviewed these changes.

Reply:
Oh dear..... I thought I got them all. Thank you!

Reply:
Commits squashed. Thanks for the review!